AlwaysOn/Wearable/src/androidTest/java/com/example/android/wearable/wear/alwayson/MainActivityTests.kt

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright (C) 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.example.android.wearable.wear.alwayson
+
+import android.app.Application
+import android.app.PendingIntent
+import android.content.Intent
+import android.view.KeyEvent
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.core.app.launchActivity
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import java.time.Clock
+import java.time.Duration
+import java.time.Instant
+import java.time.ZoneId
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.TestCoroutineDispatcher
+import org.junit.After
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@OptIn(ExperimentalCoroutinesApi::class)
+class MainActivityTests {
+    private val context = ApplicationProvider.getApplicationContext<Application>()
+
+    private val testDispatcher: TestCoroutineDispatcher = TestCoroutineDispatcher()
+
+    /**
+     * A timestamp in the relatively far, far future (year 2200).
+     *
+     * This ensures the real alarm manager won't actually trigger.
+     */
+    private var instant = Instant.ofEpochMilli(7258118400000)
```

Comment:
Could you make this a constant so it's easier to read, e.g.,

private const val YEAR_2200 = 7258118400000

Reply:
Updated! I only defined one timestamp, and just added an offset for the other usages.